lib/node_modules/@stdlib/blas/ext/base/dapxsumkbn/test/test.dapxsumkbn.js

```diff
@@ -110,7 +109,7 @@ tape( 'the function supports a `stride` parameter', function test( t ) {
 		2.0
 	]);
 
-	N = floor( x.length / 2 );
+	N = 4;
 	v = dapxsumkbn( N, 5.0, x, 2 );
```

Comment:
```suggestion
	v = dapxsumkbn( 4, 5.0, x, 2 );
```

Remove `N` declaration